docs/developer/releasing.md

```diff
@@ -11,14 +11,14 @@
 
 1. The container image is found in the registry `gcr.io/k8s-cluster-api` with an
    image name of `cluster-api-controller` and a tag that matches the release
-   version. For example, in the `0.0.0-alpha.4` release, the container image
-   location is `gcr.io/k8s-cluster-api/cluster-api-controller:0.0.0-alpha.4`
+   version. For example, in the `v0.1.3` release, the container image
+   location is `gcr.io/k8s-cluster-api/cluster-api-controller:0.1.3`
```

Comment:
Wouldn't the container image tag also be `v0.1.3`?

Reply:
Maybe? I'm not really partial either way.

Reply:
Given that it matched so far, I'd prefer if we keep it consistent going forward